resources/views/auth/login.blade.php

```diff
@@ -57,15 +57,12 @@
                 </label>
               </div>
 
-              @if (isset($errors))
-                @if (count($errors) > 0)
-                  <div class="form-group links">
-                    <ul>
-                      <li>{{ trans('auth.password_forget') }}&nbsp;<a href="{{ route('password.request') }}">{{ trans('auth.password_reset') }}</a></li>
-                    </ul>
-                  </div>
-                @endif
-              @endif
+              <div class="form-group links">
```

Comment:
This is just a nice to have. I don't see why we should hide this link unless we make an error.

Reply:
OK why not